web/hosts_next.go

```diff
@@ -34,12 +50,40 @@ func NewHostListNextHandler(hostsService services.HostsNextService) gin.HandlerF
 		pagination := NewPaginationWithStrings(len(hostList), page, perPage)
 		firstElem, lastElem := pagination.GetSliceNumbers()
 
+		hContainer := NewHostsNextHealthContainer(hostList)
+		hContainer.Layout = "horizontal"
+
 		c.HTML(http.StatusOK, "hosts_next.html.tmpl", gin.H{
-			"Hosts":          hostList[firstElem:lastElem],
-			"AppliedFilters": query,
-			"FilterSIDs":     filterSIDs,
-			"FilterTags":     filterTags,
-			"Pagination":     pagination,
+			"Hosts":           hostList[firstElem:lastElem],
+			"AppliedFilters":  query,
+			"FilterSIDs":      filterSIDs,
+			"FilterTags":      filterTags,
+			"Pagination":      pagination,
+			"HealthContainer": hContainer,
 		})
 	}
 }
+
+type SendHeartBeat struct {
+	AgentID string `json:"agent_id"`
+}
```

Comment:
How about renaming `SendHeartBeat` to `SentHeartBeat`? Or just `HeartBeatPayload`? 'Cause `SendHeartBeat` sounds more like an imperative verb

Reply:
I will remove this soonish after addressing Nelson comments!